### PR TITLE
nondefault copyright in packaging step

### PR DESCRIPTION
**Pull request by zanderz**
Jack pointed out that someone pointed out that the copyright said Github
image

### CODE REVIEW

File: shared/desktop/package.js [hunk 1]
```diff
@@ -180,22 +182,6 @@ function log (plat, arch) {
       console.error(err)
       process.exit(1)
     }
```
Comment (chrisnojima) [inline shared/desktop/package.js]: Why did this get deleted

**Comment by keybase-ci-visdiff** [comment]
The commits 70e378a9e6f6...9963abbb6ddc introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **New_Proof_Post-DNS** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1487705875406-70e378a9e6f6-9963abbb6ddc-linux/New_Proof_Post-DNS-70e378a9e6f6.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1487705875406-70e378a9e6f6-9963abbb6ddc-linux/New_Proof_Post-DNS-9963abbb6ddc.png)  
   New_Proof_Post-DNS comparison
</summary>

**Comment by zanderz** [comment]
whoops, sorry about the merge error.

**Comment by keybase-ci-visdiff** [comment]
The commits 70e378a9e6f6...773a7c340663 introduce visual changes on linux. <details><summary>:mag_right: 1 changed</summary>

 * Changed: **New_Proof_Post-DNS** [(before)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1487706550215-70e378a9e6f6-773a7c340663-linux/New_Proof_Post-DNS-70e378a9e6f6.png) [(after)](https://keybase-jenkins-visdiff.s3.amazonaws.com/1487706550215-70e378a9e6f6-773a7c340663-linux/New_Proof_Post-DNS-773a7c340663.png)  
   New_Proof_Post-DNS comparison
</summary>

**Pull request merged**